src/index.html

```diff
@@ -44,335 +43,472 @@
-                                <div class="checkbox">
-                                    <label data-toggle="collapse" href="#collapseConfirmations" aria-expanded="false" aria-controls="collapseConfirmations">
-                                        <input type="checkbox" id="confirmationsCbx" value="confirmations" aria-label="CONFIRMATIONS"> CONFIRMATIONS
-                                        <span class="fa-layers fa-fw" data-toggle="tooltip" data-placement="top" title="Tootltip informations">
-                                            <i class="fas fa-question-circle"></i>
-                                        </span>
-                                    </label>
+                            <div class="carousel-item sonar-settings">
+                                <h3 class="brown">CHOOSE KIND OF SONAR</h3>
+                                <div class="selected-informations">
+                                    <div>
+                                        <span class="selected-blockchain"></span>
+                                        <span class="selected-tag"></span>
+                                    </div>
+                                    <div id="choosen-address">
+                                    </div>
                                 </div>
-                                <div class="collapse" id="collapseConfirmations">
-                                    <div class="card card-body">
-                                        <div class="radio-buttons">
-                                            <div>
-                                                <label>
-                                                    <input type="radio" name="confirmationsOptions" id="radioValidationEach" value="radioValidationEach"> GET NOTIFIED FOR EACH CONFIRMATION UNTIL VALIDATION (7 CONFIRMATIONS)
+                                <div>
+                                    <i class="fas fa-chevron-down fa-2x"></i>
+                                </div>
+
+                                <div class="setting-rows">
+                                    <div class="row">
+                                        <div class="col-md-4"></div>
+                                        <div class="form-group col-md-4">
+                                            <div class="sonar-checkbox">
+                                                <label class="switch">
+                                                    <input type="checkbox" id="incomingTransactionCbx" value="incoming-transaction" aria-label="INCOMING TRANSACTION">
+                                                    <span class="slider round"></span>
                                                 </label>
+                                                INCOMING TRANSACTION
+                                                <span class="fa-layers fa-fw" data-toggle="tooltip" data-placement="top" title="Tootltip informations">
```

Comment:
Do we show tooltip icons even if it does nothing on hover ?

Reply:
I'm waiting for tooltips